test/views/DepartureSummaryViewSpec.scala

```diff
@@ -72,6 +72,16 @@ class DepartureSummaryViewSpec extends UnitViewSpec with Stubs with Injector {
       departureSummaryViewOut.getElementById("title").text() mustBe messages("summary.departure.title")
     }
 
+    "have correct back link for depart out of uk" in {
+
+      departureSummaryViewOut.getElementById("link-back").attr("href") must endWith("/transport")
```

Comment:
Small comment, for url you can use `routes.controllers.TransportController.displayPage().url`

Reply:
so in that case we can use mustBe not must endWith, but actual approach is correct too

Reply:
Yes.  That would be better.    I copied a similar test.